### ISSUE TITLE
yolox_x yields -1.000 mAP over many epochs

### ISSUE THREAD

**Issue by ajtao** [ifzhang/ByteTrack#270]
I'm running the yolox_x_mix_det.py recipe with the exact training command listed in the README and getting -1.000 for mAP's. I've trained for 14 epochs so far. The loss has fallen a little but is in the 2-3 range.

I was able to get reasonable mAPs from yolox_s_mix_det.py but yolox_x_mix_det uses the train.json as the validation set, so i'm not sure that's a fair comparison.

Clearly something is wrong, but yet I see lots of folks about to train these models. What am i doing wrong?

**Comment by L00qqq** [comment]
I also encountered the same problem, what is the reason?
